test/IECoreMaya/FromMayaPlugConverterTest.py

```diff
@@ -147,9 +147,23 @@ def testVectorArrayData( self ) :
 		self.assert_( converted.isInstanceOf( IECore.V3dVectorData.staticTypeId() ) )
 
 		for point, index in itertools.product( xrange( 2 ), xrange( 3 ) ):
-		    self.assertAlmostEqual( converted[ point ][ index ], data[ point ][ index ] )
+			self.assertAlmostEqual( converted[ point ][ index ], data[ point ][ index ] )
 
 		self.assertEqual( converted.getInterpretation(), IECore.GeometricData.Interpretation.Vector )
 
+	def testMultiPlug( self ):
+		s = maya.cmds.polySphere()[0]
+		bs = maya.cmds.blendShape()[0]
+		multi = bs + '.inputTarget[0].paintTargetWeights'
+		maya.cmds.setAttr(multi + '[3]', 8)
+		maya.cmds.setAttr(multi + '[5]', 9)
+
+		converter = IECoreMaya.FromMayaPlugConverter.create('blendShape1.inputTarget[0].baseWeights')
+		self.assert_( converter )
+		data = converter.convert()
+
+		for index, value in zip(data['indices'], data['data']):
```

Comment:
Given this test passed even with the previous mistake, maybe we should make it more explicit and make sure we have 3,5,8,9 in all the spots we expect.